play/src/front/Phaser/Game/GameScene.ts

```diff
@@ -1870,6 +1952,36 @@ export class GameScene extends DirtyScene {
                     warningMessageStore.addWarningMessage(errorMessage.message);
                 });
 
+                // The proximityPrivateMessageToClientMessageStream is completed in the RoomConnection. No need to unsubscribe.
+                //eslint-disable-next-line rxjs/no-ignored-subscription, svelte/no-ignored-unsubscribe
+                this.connection.proximityPrivateMessageToClientMessageStream.subscribe(
+                    (proximityPrivateMessageToClientMessage) => {
+                        console.info("proximity private message not implemented yet!");
+                    }
+                );
+
+                // The proximityPublicMessageToClientMessageStream is completed in the RoomConnection. No need to unsubscribe.
```

Comment:
@gparant 
I think you forgot to add those stores to the "completeStreams()" section of RoomConnection, so they are in fact leaking :fearful: (see my comment above)